traitsui/testing/tester/editor_action_registry.py

```diff
@@ -0,0 +1,119 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+
+from traitsui.testing.tester.abstract_registry import AbstractRegistry
+from traitsui.testing.tester.exceptions import ActionNotSupported
+
+
+class EditorActionRegistry(AbstractRegistry):
```

Comment:
Naming things is _still_ hard: "Action" is an overloaded thing in TraitsUI - in particular there is a potential for confusion with menu/toolbar `Action`s.  "Interaction" may be a better word to use throughout?